modules/standard/Random.chpl

```diff
@@ -89,12 +128,87 @@ record SeedGenerators {
   }
 };
 
-const SeedGenerator: SeedGenerators;
+
+// CHPLDOC FEEDBACK: If easy, I'd suggest either deprecating the 
+// :arg <type> <name>: form or else switching the order to 
+// :arg <name> <type>: as it's an easy trap to simply type the same
+// argument as in the function's signature.
+
+
+// CHPLDOC FIXME: int(64) prints out as int(64)(64)
+
+// CHPLDOC FIXME: if the first line below is shifted one character to the
+// left, it ends up being rendered like a method
+
+// CHPLDOC FIXME: if the first line below is shifted one character to the
+// right, it ends up causing a warning (promoted to error) in the .rst
+// file.  It'd be preferable to have it declare the issue in terms of
+// the .chpl line numbers.
+
+/*
+  Fill an array of real(64), imag(64), or complex(128) elements with
+  pseudorandom values in parallel using a new
+  :chpl:class:`RandomStream` created for the lifetime of this routine.
+  The first `arr.size` values from the stream will be assigned to the
+  array's elements in row-major order for `real` and `imag` elements.
+  For complex elements, consecutive pairs of random numbers are
+  assigned to the real and imaginary components, respectively.  The
+  parallelization strategy is determined by the array.
+
+  :arg arr: The array to be filled, where T is real(64), imag(64), or complex(128).
+  :type arr: [] T
+
+  :arg seed: The seed to use for the PRNG.  Defaults to :chpl:const:`SeedGenerator`.currentTime.
```

Comment:
I wrestled with this a bit and took the approach here because I thought it would result in a valid hyperlink once SeedGenerator made it into the rst.  I was guessing that SeedGenerator.currentTime would not since it would require sphinx to make a leap from SeedGenerator (the variable) to SeedGenerators (the record).

To simplify, if I have:

record R {
  proc foo() ...
}

var myR: R;

how would we want to refer to myR.foo()?  I was betting that a hyperlink to myR followed by foo would be better than a single hyperlink to myR.foo() that didn't point anywhere.  Though in retrospect, perhaps I could make the myR part hyperlink to the record and the foo() part hyperlink to R.foo?  (i.e., two hyperlinks rather than one?)

(Of course, if we had static methods, this case would be much cleaner... though others would still run into this issue).